armi/settings/fwSettings/globalSettings.py

```diff
@@ -604,12 +605,20 @@ def defineSettings() -> List[setting.Setting]:
             label="Clean up Files at BOC",
             description="Clean up files at the beginning of each cycle (BOC)",
         ),
+        setting.Setting(
+            CONF_STATIONARY_BLOCK_FLAGS,
+            default=["GRID_PLATE"],
+            label="stationary Block Flags",
+            description="Blocks with these flags will not move in moves. "
+            "Used for fuel management.",
+        ),
         setting.Setting(
             CONF_STATIONARY_BLOCKS,
             default=[],
             label="stationary Blocks",
-            description="blocks with these indices (int values) will not move in "
-            "moves.",
+            description="Blocks with these indices (int values) will not move in "
+            "moves. Used for fuel management. "
+            "Deprecated setting, use CONF_STATIONARY_BLOCK_FLAGS",
```

Comment:
I think we may as well remove this now that it is being replaced.

Reply:
Removed the setting.

Reply:
Awesome! This will affect some downstream repos.